src/components/GuppyWrapper/index.jsx

```diff
@@ -10,7 +10,7 @@ import {
   getAccessibleResources,
   askGuppyForSubAggregationData,
 } from '../Utils/queries';
-import { ENUM_ACCESSIBILITY, FILE_FORMATS } from '../Utils/const';
+import { ENUM_ACCESSIBILITY } from '../Utils/const';
```

Comment:
If we are not going to use `FILE_FORMATS` might be better to just delete it from `const.js`